### ISSUE TITLE
Responding to Auction Created (rename this)

### ISSUE THREAD

**Issue by NullCase** [RoboMWM/GPAuctions#7]
Once an auction is created.

1) Convert the claim to Admin Claim. 
_(Prevents someone from selling the claim mid auction, removing all of its valuables_
2) Deduct the claim blocks spent on that claim from the owner/former owner. 
_(this prevents the exploit we solved in GPRealEstate)._
3) Update the sign information as described here: https://github.com/NullCase/GPRealEstate/issues/1#issuecomment-459858952

What did I forget to list?

**Comment by RoboMWM** [comment]
I thought about the whole "convert to admin claim thing" and I don't really see any reason why this would need to happen:

- When auctions happen in real life (especially online) the item is still the owner's until the auction actually closes with a winning bid (and payment is made).
- If the player wants to provide a "buyout" price via also selling with GPRealEstate, I don't see why he should be prevented from doing so (akin to "Buy it now" price on ebay and such).

When auction ends, would be easy to see if claim still belongs to auction starter. Or can monitor GPR's "sell" event (which would have to be added) and cancel auction on a sale.

**Comment by RoboMWM** [comment]
But if you still want the "auction house" design where the seller can't do anything with the property after "selling" it or exchanging it with the "auction house," then IMO it should behave as such - player sells property to "admin claim," and after auction concludes he should receive payment. If nobody bids, then the property just becomes unclaimed.

That's just the two designs I see IMO. But if you insist on your original design that can work - it's just not intuitive IMO and definitely more work, since claim blocks have to be properly tracked and reimbursed afterwards.

**Comment by NullCase** [comment]
It sounds like you would design the auction system a bit differently.

**Comment by NullCase** [comment]
Do you feel like I am changing the design without compensating you for it's cost?

**Comment by NullCase** [comment]
OK, your original compensation for this work was $115. It seems like extra things have popped up.

What if I increase your compensation to $131 for Auctions?

Because I want to cover any more work that appears while we put together this update. 

Four payments of $15 and a $71 bonus upon final completion / after testing (as usual).

**Comment by RoboMWM** [comment]
Yea, will have to manage more things. Also, the trustlist would have to be wiped. Would you want that to be preserved as well if the auction doesn't get any bids, or just have it wiped on auction start?

**Comment by NullCase** [comment]
Yeah, just clear any /trustlist permissions. The guys can redo them as necessary if the auction doesn't occur. The extra work to keep them I don't think is worth it.

**Comment by RoboMWM** [comment]
Ok, pushed a beta build. I changed how the config is accessed, so it may say something about the old signHeader node. Sign updating and bidding was implemented, as well as end-of-auction #3 . (Probably should add a way to force-end an auction, or to specify auction time/default auction time to easily test this.)

Reopen and/or comment as needed, or just create individual new issues, depending if bug or design issue.
